8/debian9/8.0/docker-entrypoint.sh

```diff
@@ -76,7 +81,7 @@ _check_config() {
 # latter only show values present in config files, and not server defaults
 _get_config() {
 	local conf="$1"; shift
-	"$@" --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null | awk '$1 == "'"$conf"'" && $3=="" { print $2 }'
+        "$@" --verbose --help 2>/dev/null | grep "^$conf" | awk '$1 == "'"$conf"'" { print $2; exit }'
```

Comment:
Please revert tabs (to keep one conversion in the whole file).

Could you explain why we need to change this? Also, could you update the PR description with the explanation about the change for posterity? We would like to track all changes.

Reply:
We need this changes because old way to get socket value from mysqld starts work wrong with 
new version of package. New version of _get_config function always return one line. 

Example output of old and new version:

root@9a92f724a4bf:/# export conf="socket"
root@9a92f724a4bf:/# /usr/sbin/mysqld --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null | awk '$1 == "'"$conf"'" && $3=="" { print $2 }'
creation.
creation.
/var/run/mysqld/mysqld.sock
root@9a92f724a4bf:/# /usr/sbin/mysqld --verbose --help 2>/dev/null | grep "^$conf" | awk '$1 == "'"$conf"'" { print $2; exit }'
/var/run/mysqld/mysqld.sock

Reply:
This looks good to me, however why did you remove `--log-bin-index="$(mktemp -u)"`?

I think we can also remove `$1 == "'"$conf"'"`. What do you think about below code?

```shell
mysqld --verbose --help 2>/dev/null | grep "^$conf" | awk '{ print $2; exit }'
```

Reply:
--log-bin-index="$(mktemp -u)" is returned to _get_config()

grep "^$conf" means sting starts with $conf. but it is can be also $conf+something. so we need to double check  hear. 
In older version we field because awk '$1 == "'"$conf"'" means it can be also ^\s$conf or ^\t$conf. but it also means it can be ^$conf+something.